app.js

```diff
@@ -2,25 +2,44 @@ const express = require("express");
 const bodyParser = require("body-parser");
 const sql = require("mssql");
 const cors = require('cors');
-const app = express(); 
+const app = express();
 
 // the classesQueries will hold all the functions handling SQL requests to the Classes table.
 const classesQueries = require("./queries/classes");
 // the cancellationRequestsQueries will hold all the functions handling SQL requests to the CancellationRequests table.
 const cancellationRequestsQueries = require("./queries/cancellation_requests");
+// the classesQueries will hold all the functions handling SQL requests to the Classes table.
+const courseRequestQueries = require("./queries/course_requests");
 
 // import the dbConfig object from another file where we can hide it.
 const dbConfig = require("./logins")
 
 // Body Parser Middleware
-app.use(bodyParser.json()); 
+app.use(bodyParser.json());
 app.use(cors());
 
 //Setting up server
- const server = app.listen(process.env.PORT || 8080, function () {
+const server = app.listen(process.env.PORT || 8080, function () {
     const port = server.address().port;
     console.log("App now running on port", port);
- });
+});
+
+/**
+ * Gets all new course requests and updates them to pending status.
+ * Link: http://localhost:8080/new_course_requests
+ */
+app.get("/new_course_requests", function (req, res) {
+    
+    sql.connect(dbConfig, function (err) {
+        if (err) console.log(err);
+
+        courseRequestQueries.getNewCourseRequests(sql, res);
+
+        courseRequestQueries.updateCourseRequests(sql);
```

Comment:
I think the update function should be called as a callback of the previous one for 2 reasons:
- It may run before the answers are even retrieved if the server is slow
- It will run even if the first one failed